### ISSUE TITLE
Refactor messages arraylist into HashMap implementation

### ISSUE THREAD

**Issue by JamesC159** [Saylux/Project3_CS4348_OpSys_Sockets#1]
I think that it would be easier to deal with a hash map rather than an array list. Just personal suggestion

**Comment by Saylux** [comment]
yeah, definitely, do you want me to do that? I just didn't think about that structure.

**Comment by JamesC159** [comment]
You don't necessarily have to. Just a thought. That is usually my go to data structure for mappings
